README.md

```diff
@@ -18,11 +20,23 @@
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+To get a local copy up and running steps.
+
+Setup If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+
+Download the ZIP from this location, or run the following git command to clone the files to your machine:
+
+git clone https://github.com/ecekpo/LEADERBOARD.git
+
+Once the files are on your machine, open the Leaderboard folder in Visual Studio Code, and follow the steps in Installation & Set Up 🛠 Installation & Set Up Install Node Modules
 
+npm install Start the development server
 
-To get a local copy up and running follow these simple example steps.
+npm start
+
+Open http://localhost:8080 to view it in the browser.
+
+🚀 Building and Running for Production Generate a full static production build
 
```

Comment:
- [ ] Your getting started section should have all the commands which are needed to start your application. I know you have added them but please try to add them as shown in this [example](https://github.com/HAMMAS-SALEEM/Leaderboard/blob/development/README.md#getting-started) :blush: